.github/workflows/deploy_tre.yml

```diff
@@ -22,13 +22,15 @@ jobs:
     uses: ./.github/workflows/deploy_tre_reusable.yml
     with:
       ciGitRef: ${{ github.ref }}
+      ACR_NAME: ${{ secrets.ACR_NAME }}
```

Comment:
This is a globally unique acr - do we really want that displayed in our logs? I toyed with this in the past, but I think it's better that the public dont see it.

Reply:
So if it's still masked, what benefit does this change have?

Reply:
For builds on `main`, none. But the `deploy_tre_reusable.yml` is also used from the PR builds, so making it an input there reduces masking on PR builds (where `ACR_NAME` points to the specific instance for the PR deployment)